lib/wallaby/experimental/chrome.ex

```diff
@@ -232,4 +239,12 @@ defmodule Wallaby.Experimental.Chrome do
       []
     end
   end
+
+  defp put_unless_nil(map, key, value) do
+    if value do
+      Map.put(map, key, value)
+    else
+      map
+    end
```

Comment:
this is totally nitpicking and not sure if others agree, but I'd implement this using 2 methods and pattern matching on nil in the first clause :)